earthpy/tests/test_spatial.py

```diff
@@ -290,4 +303,51 @@ def test_crop_image_fails_empty_list(basic_image_tif, basic_geometry):
         es.crop_image(list(), basic_geometry)
     with rio.open(basic_image_tif) as src:
         with pytest.raises(ValueError):
+
             es.crop_image(src, list())
+
+
+def test_hillshade_shape(hillshade_arr):
+    """If provided with input array not of shape (rows, columns),
+    ValueError returned."""
+
+    # Test data
+    arr = hillshade_arr
+
+    # Check ValueError
+    with pytest.raises(
+        ValueError, match="Input array should be of shape: rows, columns"
+    ):
+        es.hillshade(arr, azimuth=315, angle_altitude=45)
+
+
+def test_hillshade_altitude(hillshade_arr):
+    """If provided with altitude value greater than 90,
+    ValueError returned."""
+
+    # Test data
+    arr = hillshade_arr
```

Comment:
you can remove this line above and just call
`hillshade_arr.squeeze` ... atleast i'm fairly certain that method will be accessible that way.

Reply:
Of course! This change has been made.